kit/popups/src/Menu/UIMenuAction.tsx

```diff
@@ -11,7 +11,7 @@ export const UIMenuAction: React.FC<UIMenuActionProps> = ({
         <UIForeground.Container key={title}>
             <UIForeground.PrimaryColumn>
                 <UIForeground.ActionCell
-                    testID={`${title}_menu_action_button`}
+                    testID={`${title}_action_button`}
```

Comment:
What do you think about changing the name of the ID in your tests, instead of applying the change here? I think that `menu_action_button` is a better suffix than just `action_button`

Reply:
it actually can be confused with UIActionButton I guess...

Reply:
@Khvoya if you want to make it simple in test, how about changing it into?
```suggestion
                    testID={`${title}_action_button_menu`}
```

In that case you will only have to add `_menu` suffix to your constants on the desired screen

Reply:
I did it because we have two similar modal menus with different item ids. For detox it's the same component and I led it to the single id

Reply:
I understand nothing, but I guess that's why I always suggest to pass testIDs from user land